package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "webpack",
     "watch": "webpack -w",
-    "start": "npm run build && node app.js",
+    "start": "npm run build && node --use_strict app.js",
```

Comment:
nice touch, but in my experience, sooner or later you'll install a dependency that has some deeply hidden dependency not compatible with strict mode... sad but true